packages/devtools_shared/test/extensions/extension_activation_test.dart

```diff
@@ -0,0 +1,110 @@
+// Copyright 2023 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'dart:io';
+
+import 'package:devtools_shared/devtools_extensions.dart';
+import 'package:devtools_shared/src/extensions/extension_activation.dart';
+import 'package:test/test.dart';
+
+void main() {
+  group('$DevToolsOptions', () {
+    late DevToolsOptions options;
+    late Directory tmpDir;
+
+    const tmpPath = '_tmp';
+    final tmpUri = Uri.parse(tmpPath);
+
+    setUp(() {
+      options = DevToolsOptions();
+      tmpDir = Directory(tmpPath)..createSync();
```

Comment:
Why not just [`Directory.createTempSync()`](https://api.dart.dev/stable/3.0.7/dart-io/Directory/createTempSync.html)?